### DISCUSSION TITLE
How to express metabolizing enzymes in PK-SIM for Vmax, Km data by HLM

### DISCUSSION THREAD

**Discussion by Cheruvh** [Open-Systems-Pharmacology/Forum#1433] [Questions & Problems]
Hi,

I have to include saturation kinetics in my model rather than linear kinetics. So I got Vmax, and Km data by Human Liver Microsomes in vitro and do not have recombinant CYP-based data. Now, how should I express the HLM enzymes under 'Metabolizing enzymes' for metabolism? 

Does mentioning HLM in the metabolizing enzyme work I tried but no enzymes were expressed in the individual and cannot see anything for the metabolism process during the simulation.
![image](https://user-images.githubusercontent.com/103236850/211692597-12e7721b-50db-43fe-b373-e1b46a34b4ce.png)

I know it is easier to express them if we have individual recombinant CYP data but no idea related to HLM. Moreover, how to express all CYP enzymes in the individual? Please explain how to do this. Thanks in advance

**Comment by tobiasK2001** [comment]
Hi,
you have to add the metabolizing enzymes to your individual first. Than you can connect them to the metabolizing processes of the compound. Please check the documentation and here could be a good start: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-expression-profile#the-workflow.
Best, Tobias

**Reply by Cheruvh** [reply]
Hi Tobias,

Thank you for the link. I read the description but was unable to find an answer to my question. I simplified my query here. 
We have a down arrow for "Metabolizing enzyme" i.e the first box in the below screenshot for 'adding an enzyme' for any compound. So what should be added/considered for a group of enzymes or pooled enzymes like Human liver microsomes (HLM) in this first box? As I have an in vitro study with Vmax and Km results using HLM, however, I cannot express them all together in the individual or population as suggested. I know this will be easy if I have the data from recombinant CYP enzymes or isozymes as the PK-SIM protein expression database accounts for each enzyme and its expression location in each organ based on rt-PCR data. On the other hand, does PK-SIM account for HLM enzymes automatically in the individual without using the protein expression database? How to do a simulation workflow with HLM or pooled enzymes together. I tried to search from HLM in the PK-SIM protein expression database but they seem to be different like HLM2 etc., Please suggest. Thank you.
![image](https://user-images.githubusercontent.com/103236850/212264637-d2667cdc-42e9-4381-b3bd-12ba013fa8e2.png)

**Comment by tobiasK2001** [comment]
Hi and Sorry for slow answering. Now I see your point and you cannot define microsomal CL for pooled enzymes here. this place is only for single enzymes. If you can calculate an in vitro intrinsic clearance from your microsomal experiment, scale it to an hepatic intrinsic clearance. Than you can transform the  hepatic intrinsic clearance to PK-Sim specific clearance and input it directly as total hepatic clearance process e.g. here:  
![image](https://user-images.githubusercontent.com/25585248/213553380-c9cb9db9-d196-4cf7-a7ee-ab1b3b01005f.png)
N.B. this is an example for rat. You might have to adapt the species. Hope this might help you.
Best, Tobias

**Reply by Cheruvh** [reply]
Hi Tobias,
No problem, thank you for your reply.
I did similarly for linear metabolism and it is working. However, I need to incorporate non-linear or Michaelis Menten Kinetics data here using HLM where we have Vmax and Km data. So how can we use this type of data for dose-escalation studies to bring saturation metabolism kinetics into the simulation? How can we express the pooled enzymes like HLM in either Individual/Population or in an Enzyme expression building block?

**Comment by tobiasK2001** [comment]
Hi, Are you really shure you need a saturable process here? Normally fisrt order works in most cases. But if you want to input Km an Vmax that would be the way:
include some enzyme in your individual BB
![image](https://user-images.githubusercontent.com/25585248/213648054-8deced99-d067-475e-83c3-b74ccb52e9f9.png)
call it at the compound BB
and choose a saturable process e.g. 
![image](https://user-images.githubusercontent.com/25585248/213651124-3c1e7798-daa1-4a12-bfec-13b93222217d.png)
In principle you should inform the process now by your enzyme concentration. But normally you don't have this info and especially for pooled enzymes. So just leave the PK-sim default values and fit Kcat to your data. That should give a reasonable description including the info for Vmax and enzyme expresse. The thinking behind this is described here:
https://pubmed.ncbi.nlm.nih.gov/22293118/
Keep in mind that this is now, if you do so, not a pure bottom up approach anymore, as you lack knowledge about expression of you pooled enzyme. But you can overcome this lack of knowledge by fitting to plasma date in a middle out fit like fashion.

Hope that might help you.

Best, Tobias

**Comment by songsong02** [comment]
I also encountered this problem and would like to know your solution. Thank you.
